pkg/apis/controlplane/v1beta2/types.go

```diff
@@ -325,6 +325,16 @@ type NodeStatsSummary struct {
 	AntreaClusterNetworkPolicies []NetworkPolicyStats `json:"antreaClusterNetworkPolicies,omitempty" protobuf:"bytes,3,rep,name=antreaClusterNetworkPolicies"`
 	// The TrafficStats of Antrea NetworkPolicies collected from the Node.
 	AntreaNetworkPolicies []NetworkPolicyStats `json:"antreaNetworkPolicies,omitempty" protobuf:"bytes,4,rep,name=antreaNetworkPolicies"`
+	// List of PodMulticast statistics collected from the Node.
+	MulticastStats []MulticastStats `json:"multicastStats,omitempty" protobuf:"bytes,5,rep,name=multicastStats"`
+}
+
+// MulticastStats contains the multicast groups that a Pod has joined.
+type MulticastStats struct {
+	Pod *PodReference `json:"pod,omitempty" protobuf:"bytes,1,opt,name=pod"`
+
+	// List of multicast IPs that the pod has joined.
+	GroupJoinedList []IPAddress `json:"groupJoinedList,omitempty" protobuf:"bytes,2,rep,name=groupJoinedList"`
```

Comment:
Maybe rename it "GroupList"?

Reply:
Not sure whether shortening the name to `GroupList` is better or not because `GroupList` can also be miscomprehended as "list of multicast groups that the pod is sending multicast traffic".